File names with spaces break analysis

### Version

_No response_

### Reproduction steps

1. File `.github/workflows/x y.yml`
2. Run `ghlint` via the action


### Expected behavior

"just works"

### Actual behavior

```
Run # Hackaround for https://github.com/orgs/community/discussions/73309
  # Hackaround for https://github.com/orgs/community/discussions/73309
  PATH="${JAVA_HOME_17_X64}/bin:${PATH}"
  echo "${FILES}" | xargs "/home/runner/work/_temp/ghlint.jar" --verbose --sarif=ghlint.sarif.json
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    FILES: /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Delete from Google App Engine.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Dependency Review.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Deploy to Google App Engine.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Generate random data.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Heroku-check-gradle.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Heroku-check-npm.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Heroku-check.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Heroku-deps.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Heroku-pr.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Heroku.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Release from main.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Sync from source.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/map-detekt.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/map-run.yml
  /home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/map.yml
Usage: ghlint [<options>] <files>...

Error: invalid value for <files>: file "/home/runner/work/net.twisterrob.cinema/net.twisterrob.cinema/.github/workflows/Delete" does not exist.
Error: Process completed with exit code 123.
```

### Additional Context

https://github.com/TWiStErRob/net.twisterrob.cinema/actions/runs/8287105647/job/22678649808?pr=709

--- a/action.yml
+++ b/action.yml
@@ -81,4 +81,5 @@ runs:
       run: |
         # Hackaround for https://github.com/orgs/community/discussions/73309
         PATH="${JAVA_HOME_17_X64}/bin:${PATH}"
-        echo "${FILES}" | xargs "${{ runner.temp }}/ghlint.jar" --verbose --sarif=ghlint.sarif.json
+        # Known issue: file names with new lines in them will be split into multiple files.
+        echo "${FILES}" | tr '\n' '\0' | xargs -0 "${{ runner.temp }}/ghlint.jar" --verbose --sarif=ghlint.sarif.json

